test/index.test.js

```diff
@@ -26,20 +26,35 @@ describe('RxRouter', () => {
 
   it('should support multiple routes, methods and Epics', async () => {
     const router = new RxRouter()
+    const methods = [
+      'del',
+      'delete',
+      'get',
+      'patch',
+      'post',
+      'put'
+    ];
 
-    router.get('/test1', obs => obs.mapTo('get-test1'))
-    router.post('/test1', obs => obs.mapTo('post-test1'))
-    router.put('/test2', obs => obs.mapTo('put-test2'))
-    router.del('/test3', obs => obs.mapTo('del-test3'))
-    router.delete('/test4', obs => obs.mapTo('del-test4'))
+    methods.forEach((methodName) => {
```

Comment:
Please use `for (let methodName of methods)` here as well as you did below (and without semicolons).